spec/spec_helper_acceptance_local.rb

```diff
@@ -73,10 +73,14 @@ def apache_settings_hash
     apache['error_log']        = 'error_log'
     apache['suphp_handler']    = 'php5-script'
     apache['suphp_configpath'] = 'undef'
-    if (operatingsystemrelease >= 7 && operatingsystemrelease < 8) && (osfamily == 'redhat')
+    if (operatingsystemrelease >= 7 && operatingsystemrelease < 9) && (osfamily == 'redhat')
       apache['version']     = '2.4'
       apache['mod_dir']     = '/etc/httpd/conf.modules.d'
-      apache['mod_ssl_dir'] = apache['confd_dir']
+      if operatingsystemrelease == 7
+        apache['mod_ssl_dir'] = apache['confd_dir']
+      elsif operatingsystemrelease == 8
+        apache['mod_ssl_dir'] = apache['mod_dir']
+      end
     elsif operatingsystemrelease >= 8 && osfamily == 'redhat'
```

Comment:
Isn't this block dead code now? It's probably better to list this first, then fall back to `>= 7`

Reply:
Noticed my mistake whilst you were commenting. I'll tweak again though to make it a bit more readable.

Reply:
IMHO this wasn't what I meant so I submitted it as https://github.com/puppetlabs/puppetlabs-apache/pull/2043